schemas/data/stall-time.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/stall-time",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Stall Time",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/metricdefinition"
+  ],
+  "description": "Describes the duration in which the playback was stalled during playback.",
```

Comment:
This is a pattern that @trieloff  introduced a while ago and is intended to provide the option to add additional metadata about the metrics so that design tools and automated processors can have more intelligence about the specifics of each metric.

Reply:
See above